UKHomeOffice/sgmr-service
# NMSW-1380 Move 'user' files into register and sign in folders

### AC / Description of changes

Tidying up ahead of new feature builds
- move "user" files into register and sign in folder


### To Test

- go to sign in page
- > page should load
- sign in
- > you should be signed in
- sign out
- > you should be signed out
- go back to sign in page
- click on create account link
- > page should load

### Notes

---

**Remember to merge to feature branches not `master`.**
